kairon/chat/handlers/channels/voice.py

```
@@ -140,6 +199,19 @@ async def _get_reprompt(self, sender_id: Text, config: dict) -> List[str]:
+    async def _record_welcome_conversation(self, sender_id: Text, welcome_msg: Text, metadata: dict, form: dict) -> None:
+        try:
+            ChannelLogs(
+                type=ChannelTypes.VOICE.value,
+                status="welcome_sent",
+                message_id=sender_id,
+                bot=self.bot,
+                user=self.user.get_user(),
+                data={**form, "provider": self.provider},
+            ).save()
+        except Exception:
+            pass
```

Ruff (0.15.20)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)